spec/committerFinderSpec.js

```diff
@@ -90,6 +90,17 @@ describe("committer finder", () => {
     message: "Tear it apart"
   };
 
+  const sortResponse = function(unsortedResponse) {
+    return {
+      unresolvedLoginNames: unsortedResponse.unresolvedLoginNames.sort(),
+      distinctUsersToVerify: unsortedResponse.distinctUsersToVerify
+        .map(function(name) {
+          return name.email;
+        })
+        .sort()
+    };
```

Comment:
Does this leave you with a list of just e-mails in distinctUsersToVerify as opposed to full user objects?
You've done the same to both sides of the comparisons but it feels wrong to chuck away some of the info when the function is called sort.
I'm sure there are clever functional ways of doing this (not sure what the equivalent of a Java comparator is) but a naive way to do it would be to use the list of emails created here to re-build the list of full objects.

Reply:
I think the distinctUsersToVerify will be a list of objects like this:
```
  const bobCommitter = {
    email: bobCommit.author_email,
    name: bobCommit.author_name,
    login: bobLogin.username
  };
```

Reply:
Ah yes, good spot, I missed that